File: Data Structures For Interview/Python/Basic Programs/FibonacciSeriesWithRecursion.py
n1=0
n2=1
n3=1

def printFibonacciSeries(count):
    global n1,n2,n3
    if count>0:
        n3=n1+n2
        print(" ",n3)
        n1=n2
        n2=n3
        printFibonacciSeries(count-1)

count=20
print(n1," ",n2)
printFibonacciSeries(count-2)

File: Data Structures For Interview/JavaScript/Basic Programs/FibonacciSeriesWithRecursion.js
let n1=0;
let n2=1;
let n3=1;

function printFibonacciSeries(count){
    if(count>0){
        n3=n1+n2;
        console.log(" "+n3);
        n1=n2;
        n2=n3;
        printFibonacciSeries(count-1);
    }
}

let count=20;

console.log(n1+" "+n2);
printFibonacciSeries(count-2);